packages/debugger/src/panels/kernelSources/model.ts

```diff
@@ -19,6 +21,11 @@ const compare = (a: IDebugger.KernelSource, b: IDebugger.KernelSource) => {
  * The model to keep track of the current source being displayed.
  */
 export class KernelSourcesModel implements IDebugger.Model.IKernelSources {
+  constructor() {
+    this.refresh = this.refresh.bind(this);
+    this._debouncer = new Debouncer(this.refresh, 500);
```

Comment:
It would be nicer to set a constant with explicit name rather than the timeout value.

Reply:
Done. Thx.